scripts/distribution/macOS-package/README.md

```diff
@@ -0,0 +1,172 @@
+# Concordium Node for MacOS
+
+## Prerequities
+
+- Xcode 13
```

Comment:
There was no way of making this work with the currently released XCode?

Reply:
No, unfortunately not. I even tried Xcode versions from 2019 and 2018. However, Xcode 13 should be released later this month.